crates/sui-e2e-tests/tests/object_balance_tests.rs

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use sui_json_rpc_types::{SuiExecutionStatus, SuiTransactionBlockEffectsAPI};
+use sui_macros::sim_test;
+use sui_protocol_config::ProtocolConfig;
+use sui_types::{
+    gas_coin::GAS,
+    transaction::{CallArg, ObjectArg},
+};
+use test_cluster::TestClusterBuilder;
+
+#[sim_test]
+async fn test_object_balance_withdraw_disabled() {
+    telemetry_subscribers::init_for_testing();
+
+    let _guard = ProtocolConfig::apply_overrides_for_testing(|_, mut cfg| {
+        cfg.create_root_accumulator_object_for_testing();
+        cfg.set_enable_accumulators_for_testing(false);
```

Comment:
I think this test should either use the default protocol config, or we should unconditionally abort in the native function. I'd like to enable AB on devnet, but this change wouldn't allow that until object balances are fully supported